js/src/builtin/streams/ReadableStreamInternals.cpp

```diff
@@ -76,7 +76,7 @@ using js::ReadableStream;
   if (!unwrappedReader) {
     return nullptr;
   }
-  MOZ_ASSERT(unwrappedReader->is<ReadableStreamDefaultReader>());
+  //  MOZ_ASSERT(unwrappedReader->is<ReadableStreamDefaultReader>());
```

Comment:
Should this be uncommented?

Reply:
I've removed the assertion entirely, we're using this code path for both default and BYOB streams (combining of `request` and `readIntoRequest` paths was done in previous implementation).